src/components/modals/FlipInputModal2.tsx

```diff
@@ -0,0 +1,301 @@
+/* eslint-disable react-native/no-raw-text */
```

Comment:
Per Sam's suggestion, we generally prefer to put these inline, right next to the affected area.

Reply:
Another copy/paste from FlipInputModal. Will fix